### PR TITLE
Runtime Runtime Fix

### PR DESCRIPTION
**Pull request by Geevies**
* Fixed null emergency shuttles causing runtime errors when the game checks for status displays.

### CODE REVIEW

**Comment by Geevies** [comment]
!bugfix

**Comment by Geevies** [comment]
!review

File: code/controllers/subsystems/emergency_shuttle.dm [hunk 1]
```diff
@@ -225,11 +225,13 @@ var/datum/controller/subsystem/emergency_shuttle/emergency_shuttle
 
 //returns 1 if the shuttle is not idle at centcom
 /datum/controller/subsystem/emergency_shuttle/proc/online()
+	if (isnull(shuttle?.location))
+		return FALSE
```
Comment (NonQueueingMatt) [inline code/controllers/subsystems/emergency_shuttle.dm:229]: huh what, all you need to do is check if the shuttle is null

Comment (Geevies) [reply code/controllers/subsystems/emergency_shuttle.dm:229]: kk

**Pull request merged**